components/content_settings/renderer/brave_content_settings_agent_impl.h

```diff
@@ -99,9 +95,6 @@ class BraveContentSettingsAgentImpl
   // cache blocked script url which will later be used in `DidNotAllowScript()`
   GURL blocked_script_url_;
 
-  // temporary allowed script origins we preloaded for the next load
-  base::flat_set<std::string> preloaded_temporarily_allowed_scripts_;
```

Comment:
this doesn't work on reloads, an embedded script is executed before this is filled.